.azure-pipelines/test_plan.py

```diff
@@ -331,12 +332,20 @@ def poll(self, test_plan_id, interval=60, timeout=36000, expected_states=""):
         help="The asic number of dut"
     )
     parser_create.add_argument(
-        "--access-token",
+        "--azp-access-token",
         type=str,
-        dest="access_token",
+        dest="azp_access_token",
         default="",
         required=False,
-        help="Authorization token to access internal resource (image, etc)"
+        help="A special variable that carries the security token used by the running build"
```

Comment:
Recommend: Token to download the artifacts of Azure Pipelines

Reply:
fixed.